docs/pages/bare/using-expo-cli.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Using Expo CLI
+description: Learn how and why you can use Expo CLI instead of @react-native-community/cli in any React Native project.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+To use the local Expo CLI in a project created with `npx react-native init`, you will first need to install the `expo` package. This package includes the Expo Modules API and Expo CLI. For a complete explanation of how to install this, see the ["Install Expo modules" guide](/bare/installing-expo-modules.md). For most apps, it's as simple as running the following command in your project directory:
+
+<Terminal cmd={['npx install-expo-modules@latest']} />
+
+## Compiling and running your app
+
+Once you have the `expo` package installed, you can use the `npx expo run:ios` and `npx expo run:android` commands as alternatives to `npx react-native run-ios` and `npx react-native run-android`. You can pass in the `--device` flag to select a device/simulator to build for — this also works for any connected iOS device.
+
+## Starting the bundler
+
+`npx expo run:[android|ios]` will start the bundler/development server. You can skip that by passing in `--no-bundler` flag. Start the development server independently with `npx expo start --dev-client`.
+
+## Why we think you should use it instead of `npx react-native`
+
+Expo CLI commands provide several benefits over the similar commands in `@react-native-community/cli`, including:
+
+- Open up the Hermes debugger in one keystroke: <kbd>j</kbd>.
```

Comment:
Built-in monorepo support

Reply:
install command / versions endpoint is another good feature.

Reply:
Built-in react-devtools support (available on expo@latest). though without expo-go or expo-dev-client, user needs to manually reload their app to make the app showing in react-devtools.